doc/nrf/releases_and_maturity/migration/zephyr_migration_guide.rst

```diff
@@ -0,0 +1,10 @@
+.. _ncs_zephyr_migration:
+
+Migration notes for SDK Zephyr
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. zephyr-migration-guide::
```

Comment:
as said: this is duplicating information that is already in the Zephyr docset (will just be split between 2 guides). I think it's easier to explain this _problem_ to developers than developing a new extension.